inc/class-wpseo-rank.php

```diff
@@ -0,0 +1,41 @@
+<?php
+/**
+ * @package WPSEO\Internals
+ */
+
+/**
+ * Holder for SEO Rank information
+ */
+class WPSEO_Rank {
+
+	const BAD = 'bad';
+	const POOR = 'poor';
+	const OK = 'ok';
+	const GOOD = 'good';
+	const NO_FOCUS = 'na';
+	const NO_INDEX = 'noindex';
+
+	/**
+	 * Holds the translation from seo score slug to actual score range
+	 *
+	 * @var array
+	 */
+	public static $range = array(
```

Comment:
Instead of a static array, I'd rather see this as a method, so that some error handling can be added too, for keys that don't exist etc.